src/test/java/stepdefinitions/StepDefinitions.java

```diff
@@ -1,66 +1,165 @@
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import pages.ContactUsPage;
 import pages.MobileChapterPage;
 import pages.ValoriHomePage;
+import pages.WorkAtValoriPage;
 
 import java.util.ArrayList;
-
 import static org.junit.Assert.assertTrue;
 
 public class StepDefinitions {
-    private WebDriver driver;
-
-    public StepDefinitions (){
-        this.driver = DriverManager.driver;
-    }
-
-
-    @Given( "^I am on the Valori homepage$" )
-    public void iAmOnTheValoriHomepage(){
-        ValoriHomePage valoriHomePage = new ValoriHomePage(driver);
-        valoriHomePage.clickCookieConsent();
-        assertTrue("Expecting the innovation header to be visible",valoriHomePage.isLogoVisible());
-    }
-
-    @And( "^I navigate to the Mobile Chapter page$" )
-    public void iNavigateToTheFullStackPage(){
-        ValoriHomePage valoriHomePage = new ValoriHomePage(driver);
-        MobileChapterPage mobileChapterPage = new MobileChapterPage(driver);
-        valoriHomePage.clickBusinessDropdown();
-        valoriHomePage.clickMobileInDropDOwn();
-        assertTrue("Expecting the full stack header to be visible", mobileChapterPage.isMobilePageVisibile());
-    }
-
-    @Then( "^I can read articles about the Mobile Chapter$" )
-    public void iCanReadAboutMobileChapter(){
-        MobileChapterPage mobileChapterPage = new MobileChapterPage(driver);
-        assertTrue(mobileChapterPage.areArticlesVisible());
-    }
-
-    @And( "^I contact Valori$" )
-    public void iContactValori()throws Throwable{
-        MobileChapterPage mobileChapterPage = new MobileChapterPage(driver);
-        ContactUsPage contactUsPage = new ContactUsPage(driver);
-        mobileChapterPage.clickContactUsButton();
-
-        //switch to new tab
-        ArrayList<String> tabs = new ArrayList<String>(driver.getWindowHandles());
-        driver.switchTo().window(tabs.get(1));
-        assertTrue(contactUsPage.isTitleVisible());
-    }
-
-    @Then( "^the contact us form is available$" )
-    public void theContactUsFormIsAvailable() throws Throwable {
-        //TODO IMPLEMENT THIS
-        //Verify the contact form fields are available.
-        throw new PendingException();
-    }
-}
+	private WebDriver driver;
+	private BrowserScreenshotMaker screenshotMaker;
+	private Logger logger;
+
+	public StepDefinitions() {
+		this.driver = DriverManager.driver;
+		this.screenshotMaker = new BrowserScreenshotMaker();
+		this.logger = LoggerFactory.getLogger(StepDefinitions.class);
+	}
+
+	@Given("^I am on the Valori homepage$")
+	public void iAmOnTheValoriHomepage() throws Throwable {
+		ValoriHomePage valoriHomePage = new ValoriHomePage(driver);
+
+		try {
+			valoriHomePage.clickCookieConsent();
+			assertTrue("Expecting the innovation header to be visible", valoriHomePage.isLogoVisible());
+		}
+
+		catch (AssertionError e) {
+			screenshotMaker.takeScreenShot(this.driver, "src/test/screenshots", StackTraceHelper.getMethodNameCaller());
```

Comment:
Hier ben je een beetje doorgeslagen. Ik zou hier geen try catch blokken in de step definitions willen zien als tester. Een oplossing zou zijn om met de junit api een rule toe te voegen waarmee on failure een screenshot gemaakt wordt.
http://www.easyselenium.com/how-to-capture-screenshot-for-failed-test-cases-in-selenium/